adapters/repos/db/vector/hnsw/deserializer.go

```diff
@@ -31,9 +31,9 @@ func NewDeserializer(logger logrus.FieldLogger) *Deserializer {
 
```

Comment:
whole file: see above, all node ids should be represented by `uint64`, removing the need for the conversions.